kubeclient.gemspec

```diff
@@ -27,5 +27,5 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency 'rubocop', '= 0.30.0'
   spec.add_dependency 'rest-client'
   spec.add_dependency 'recursive-open-struct', '= 1.0.0'
-  spec.add_dependency 'http', '= 0.9.8'
+  spec.add_dependency 'http', '< 3', '>= 0.98'
```

Comment:
oops!  we changed  0.9.8 to 0.98 by mistake :-(
I'm gonna check what this resolves to in practice to figure if we need another release...

Reply:
http versions are: (3.0.0, 2.2.2, 2.2.1, 2.2.0, 2.1.0, 2.0.3, 2.0.2, 2.0.1, 2.0.0, 1.0.4, 1.0.3, 1.0.2, 1.0.1, **1.0.0, 0.9.9, 0.9.8**, 0.9.7, 0.9.6, 0.9.5, 0.9.4, 0.9.3, 0.9.2, 0.9.1, 0.9.0, 0.8.14, 0.8.13, 0.8.12, 0.8.11, 0.8.10, 0.8.9, 0.8.8, 0.8.7, 0.8.6, 0.8.5, 0.8.4, 0.8.3, 0.8.2, 0.8.1, 0.8.0, 0.7.4, 0.7.3, 0.7.2, 0.7.1, 0.7.0, 0.6.4, 0.6.3, 0.6.2, 0.6.1, 0.6.0, 0.5.1, 0.5.0, 0.4.0, 0.3.0, 0.2.0, 0.1.0, 0.0.2, 0.0.1, 0.0.0)
This prevents 0.9.8 and 0.9.9 from being used, effectively requires `>= 1.0.0`.